metricbeat/module/uwsgi/status/_meta/data.json

```diff
@@ -1,53 +1,31 @@
 {
-    "@timestamp": "2011-11-11T00:00:00.000Z",
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "event": {
+        "dataset": "uwsgi.status",
+        "duration": 115000,
+        "module": "uwsgi"
+    },
     "metricset": {
-      "rtt": 3069,
-      "name": "stat",
-      "module": "uwsgi"
+        "name": "status"
+    },
+    "service": {
+        "address": "127.0.0.1:9192",
+        "type": "uwsgi"
     },
     "uwsgi": {
-      "status": {
-        "total": {
-          "requests": 102,
-          "exceptions": 1,
-          "write_errors": 0,
-          "read_errors": 0,
-          "pid": 1
-        },
-
```

Comment:
Seeing that empty line in here it definitively indicates this was generated manually. I'm glad we have a new one.

Reply:
Nice catch!